test/unit/containers/Collective.js

```diff
@@ -95,7 +95,7 @@ describe('Collective container', () => {
       expect(payment.stripeToken).to.be.equal(token.id);
       expect(payment.email).to.be.equal(token.email);
       expect(payment.currency).to.be.equal(collective.currency);
-      expect(payment.amount).to.be.equal(10);
+      expect(payment.amount).to.be.equal(1000);
```

Comment:
shouldn't we also add a e2e test for this?
just to make sure that one day we don't have an email from wwcode saying they can't add funds.